game.exs

```diff
@@ -1,7 +1,39 @@
 defmodule GuessingGame do
 
-  def hi() do
-    IO.puts("hello world")
+  def compare(n, t) do
+    cond do
+      n < t -> :low
+      n == t -> :equal
+      n > t -> :high
+    end
+  end
+
+  def print_state(:low),      do: IO.puts("Too low")
```

Comment:
I like the name `print_result` or `print_outcome` more, you're not really printing state, you're printing the result of the guess.